src/js/media-controller.js

```diff
@@ -622,155 +625,85 @@ class MediaController extends MediaContainer {
     els.splice(index, 1);
   }
 
-  // Mimick the media element API, but use it to dispatch media UI events
-  // so that everything happens through the events.
-  // Not sure how far we should take this API
-  play() {
-    this.dispatchEvent(new window.CustomEvent(MEDIA_PLAY_REQUEST));
-  }
-
-  pause() {
-    this.dispatchEvent(new window.CustomEvent(MEDIA_PAUSE_REQUEST));
-  }
-
-  get airplayUnavailable() {
-    return this._airplayUnavailable;
-  }
-
-  get paused() {
-    if (!this.media) return true;
-
-    return this.media.paused;
-  }
-
-  get muted() {
-    return !!(this.media && this.media.muted);
-  }
-
-  set muted(mute) {
-    const eventName = mute ? MEDIA_MUTE_REQUEST : MEDIA_UNMUTE_REQUEST;
-    this.dispatchEvent(new window.CustomEvent(eventName));
-  }
-
-  get volume() {
-    const media = this.media;
-
-    return media ? media.volume : 1;
-  }
-
-  set volume(volume) {
-    this.dispatchEvent(
-      new window.CustomEvent(MEDIA_VOLUME_REQUEST, { detail: volume })
-    );
-  }
-
-  get volumeLevel() {
-    let level = 'high';
-
-    if (!this.media) return level;
+  /**
+   * Media Controller should not mimic the HTMLMediaElement API.
+   * @see https://github.com/muxinc/media-chrome/pull/182#issuecomment-1067370339
+   */
+}
 
-    const { muted, volume } = this.media;
+const getPaused = (el) => {
```

Comment:
Nit: We might use `controller` for these args instead of `el`, to be clear what the expected API is. They kind of look like generic element functions at first.